plugins/yugabytedb/database_credentials.go

```diff
@@ -0,0 +1,91 @@
+package yugabytedb
+
+import (
+	"context"
+	"strings"
+	"errors"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DatabaseCredentials() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DatabaseCredentials,
+		DocsURL:       sdk.URL("https://docs.yugabyte.com/preview/admin/ysqlsh/#connect-to-a-database"), 
+		ManagementURL: sdk.URL("https://cloud.yugabyte.com/"), 
```

Comment:
Could we update the management URL to https://cloud.yugabyte.com/clusters, because it directly lists the list of clusters on the account? One can click on the cluster there to connect to it.